src/index.html

```diff
@@ -17,16 +17,35 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="app">
+      <div class="app__wrapper">
+        <form
+          class="search-bar"
+          data-qa="big"
+        >
+          <label class="search-input">
+            <input
+              type="text"
+              class="search-input__item"
```

Comment:
and this is input ;)
```suggestion
              class="search-input"
```

Reply:
check answer above